10up/classifai
# Add: service provider request response info to Site Health Info screen

### Description of the Change
Add new latest response line to each service provider in Site Health > Info > ClassifAI.


### Alternate Designs
n/a


### Benefits
Show the results of the most recent request to each provider as this will likely help when troubleshooting service provider-related issues.


### Possible Drawbacks
n/a


### Verification Process
0. Go to Tools > Site Health > Info > ClassifAI.
1. See Latest response shows `N/A`.
2. With Watson NLU enabled, update a post.
3. With Computer Vision enabled, upload an image.
4. Go to Tools > Site Health > Info > ClassifAI.
5. See the latest responses with data.


### Checklist:






### Applicable Issues
Closes #143 


### Changelog Entry


Added: Result of the latest request of all providers to Site Health.
